src/components/NewsSectionPlaceholder.tsx

```diff
@@ -0,0 +1,47 @@
+import { StackNavigationProp } from '@react-navigation/stack';
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+
+import { colors, Icon, texts } from '../config';
+import { imageHeight, imageWidth } from '../helpers';
+
+import { Button } from './Button';
+import { BoldText, RegularText } from './Text';
+import { Wrapper } from './Wrapper';
+
+type Props = {
+  navigation: StackNavigationProp<any>;
+};
+
+export const NewsSectionPlaceholder = ({ navigation }: Props) => {
+  return (
+    <View>
+      <Wrapper>
+        <Icon.EmptySection
+          color={colors.primary}
+          size={imageHeight(imageWidth())}
+          style={styles.icon}
+        />
+        <Wrapper>
+          <BoldText primary center>
+            {texts.placeholder.homeSectionTitle}
+          </BoldText>
+          <RegularText primary center small>
+            {texts.placeholder.homeSectionSubtitle}
+          </RegularText>
+        </Wrapper>
+        <Button
+          title={texts.placeholder.homeSectionButton}
+          onPress={() => navigation.push('Settings')}
+        />
```

Comment:
I adjusted the texts:
1. Keine `categoryTitle` vorhanden
2. Hast du alle Quellen abgewählt?
3. Filter ändern

my reasoning for these text is as follows:
* for the first text it makes sense to include the correct category title, as there are specifically no items of that category
* for the second and third i removed the reference to a category, as the data provider filter settings are category agnostic, both in terms of how they function as well as how they are configurable
* for the second text it seemed weird to have cases like "Meldungen-Quellen" or "Rathaus Informationen-Quellen", and as you said "Nachrichten" is not always fitting.
* this allows us to not have an additional layer of complexity when setting up homesections by reusing the category title in the first string, where it will always make sense, but leaving the other two strings independent of the category.

What are your thoughts?

Reply:
i like everything you said and did and totally agree.